src/components/FinalizarProjetoDialog.tsx

```
@@ -0,0 +1,123 @@
+} from '@/components/ui/dialog'
+import { Input } from '@/components/ui/input'
+import { Button } from '@/components/ui/button'
+import { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { toast } from 'react-toastify'
+import { AxiosError } from 'axios'
+import { ErrorResponse, Projeto } from '@/commons/interfaces'
+import ProjetoService from '@/projetos/ProjetoService'
+
+export function FinalizarProjetoDialog({ id }: { id: number }) {
+    const [open, setOpen] = useState(false)
```

ESLint

[error] Delete `··`

(prettier/prettier)